[GOV] vote for removal from CC (#3697)
sktime/sktime: docs/source/get_involved/governance.rst

Tenure also ends automatically when a CC member's tenure as core developer ends,
e.g., through resignation or inactivity.

> CC members can be removed from the CC by an 80% majority of all core developers,

Reviewer: ```suggestion
CC members can be removed from the CC by a majority of all voting core developers.
```

> i.e., non-participation abstention counts as against.

Reviewer: ```suggestion
```

This vote is anonymous and takes place on the private core developer channel.

> It requires a 5 working day (excl week-ends) discussion period, followed

Reviewer: ```suggestion
```

Author: I think this is important, people need to know this is coming. We have this for appointments, so we should also do this for revocations.
